src/app/modules/feedback/__tests__/feedback.service.spec.ts

```diff
@@ -144,11 +144,11 @@ describe('feedback.service', () => {
         expectedCount
       ).toFixed(2)
       expect(actualResult.isOk()).toEqual(true)
-      expect(actualResult._unsafeUnwrap()).toEqual({
-        average: expectedAverage,
-        count: expectedCount,
-        feedback: expectedFeedbackList,
-      })
+      expect(actualResult._unsafeUnwrap().average).toEqual(expectedAverage)
+      expect(actualResult._unsafeUnwrap().count).toEqual(expectedCount)
+      expect(actualResult._unsafeUnwrap().feedback.sort()).toEqual(
```

Comment:
actually, do we need to sort the feedback by date? Instead of running sort on the entire array

Reply:
sure, that would be more efficient